### PR TITLE
Implement QuickLinks component

### PR DESCRIPTION
**Pull request by reynard2007**
PR for Task #9 to replace PR #28

### CODE REVIEW

**Comment by vikasrohit** [comment]
Merging. Final fixes to be done later on.

File: components/QuickLinks/QuickLinks.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+require('./QuickLinks.scss')
```
Comment (nlitwin) [inline components/QuickLinks/QuickLinks.jsx]: same as comments in SearchBar/SearchSuggestions

File: components/QuickLinks/QuickLinks.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+@import "work/work-includes";
```
Comment (nlitwin) [inline components/QuickLinks/QuickLinks.scss]: Can remove, in favor of 'topcoder/tc-includes'

File: components/QuickLinks/QuickLinksExample.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+require('./QuickLinksExample.scss')
```
Comment (nlitwin) [inline components/QuickLinks/QuickLinksExample.jsx]: 'use strict'/imports

File: components/QuickLinks/QuickLinksExample.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+@import "work/work-includes";
```
Comment (nlitwin) [inline components/QuickLinks/QuickLinksExample.scss]: topcoder includes > work includes

**Pull request merged**